server/package.json

```diff
@@ -0,0 +1,17 @@
+{
+  "name": "server",
+  "version": "1.0.0",
+  "description": "Server",
+  "main": "index.js",
+  "scripts": {
+    "test": "mocha ./test/**"
```

Comment:
Maybe we should stick to jest as a testing framework?

Reply:
Sure, there is no tests so far, I will use jest when introducing them  !